src/models.ts

```diff
@@ -11,6 +11,23 @@ export interface Serp {
   hotels?: Hotels;
   adwords?: { adwordsTop?: Ad[]; adwordsBottom?: Ad[] };
   availableOn?: AvailableOn[];
+  shop?: ShopCard[];
+}
+
+export interface ShopCard {
+  imgLink: string;
+  title: string;
+  price: number;
+  currency: string;
+  shoppingSite: string;
+  description: ShopDescription;
+}
+
+export interface ShopDescription {
```

Comment:
Context wise this is not a shop description but maybe productInfo, also why move it to additional object why not just add all the properties to the main object?